Source/Csla.Shared/ApplicationContext.cs

```diff
@@ -878,8 +878,20 @@ internal static IServiceProvider ScopedServiceProvider
           return null;
       }
     }
+
+    private static int _scopeRef;
```

Comment:
I don't think this can be `static`, because this is logical (and maybe physical) server-side code. That means many data portal requests may be running at the same time, each with their own scope.

Reply:
I published a fix that use Csla.Server.DataPortal._oldLocation and do not create any variables.